x-pack/plugins/code/public/components/admin_page/project_tab.tsx

```diff
@@ -149,7 +150,10 @@ class CodeProjectTab extends React.PureComponent<Props, State> {
               <h3>Repository URL</h3>
             </EuiTitle>
             <EuiForm>
-              <EuiFormRow>
+              <EuiFormRow
+                isInvalid={this.state.repoURL.trim() === ''}
```

Comment:
let's have a `validateUrl` function and call it in here in case any future change. otherwise, you have to make changes in multiple places.